psdk_wrapper/src/psdk_wrapper.cpp

```diff
@@ -32,9 +32,11 @@ PSDKWrapper::PSDKWrapper(const std::string &node_name)
   declare_parameter("app_license", rclcpp::ParameterValue(""));
   declare_parameter("developer_account", rclcpp::ParameterValue(""));
   declare_parameter("baudrate", rclcpp::ParameterValue(""));
-  declare_parameter("hardware_connection", rclcpp::ParameterValue(""));
-  declare_parameter("uart_dev_1", rclcpp::ParameterValue(""));
-  declare_parameter("uart_dev_2", rclcpp::ParameterValue(""));
+  std::string ros_pkg_path =
+      ament_index_cpp::get_package_share_directory("psdk_wrapper");
+  std::string default_config_file = ros_pkg_path + "/cfg/link_config.json";
```

Comment:
you can also set the default from the launch file, and reduce few lines here, juts if you want to keep this with less boilerplate

Reply:
as discussed, the default path is hardcoded here ,but users have the option to change it via the link_config_file_path parameter exposed